pkgs/by-name/xm/xmltoman/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, perl538Packages
+, perl
+, installShellFiles
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "xmltoman";
+  version = "0.6";
+
+  src = fetchFromGitHub {
+    owner = "atsb";
+    repo = "xmltoman";
+    rev = version;
+    hash = "sha256-EmFdGIeBEcTY0Pqp7BJded9WB/DaXWcMNWh6aTsZlLg=";
+  };
+
+  nativeBuildInputs = [
+    perl
+    installShellFiles
+  ];
+
+  buildInputs = [
+    perl538Packages.XMLTokeParser
+  ];
+
+  dontBuild = true;
+
+  # File installation and setup similar to Makefile commands below.
+
+  installPhase = ''
+    runHook preInstall
```

Comment:
```suggestion
    runHook preInstall

```

Reply:
What im i supposed to do here they seem pretty much same

Reply:
it's just a common formatting pattern to leave a new line after `runHook preInstall`

and new line before `runHook postInstall`

Reply:
Ok understood. Thanks.